packages/imperative/src/cmd/src/CommandProcessor.ts

```diff
@@ -814,28 +805,20 @@ export class CommandProcessor {
         const configSecureProps: string[] = [];
 
         /**
-         * If using config, then we need to get the secure properties from the config
+         * Need to get the secure properties from the config
          */
-        if (useConfig) {
+        const combinedProfiles = [ ...showInputsOnly.requiredProfiles ?? [], ...showInputsOnly.optionalProfiles ?? [] ];
+        combinedProfiles.forEach((profile) => {
+            const name = ConfigUtils.getActiveProfileName(profile, commandParameters.arguments); // get profile name
+            const props = this.mConfig.api.secure.securePropsForProfile(name); // get secure props
+            configSecureProps.push(...props); // add to list
+        });
 
-            const combinedProfiles = [ ...showInputsOnly.requiredProfiles ?? [], ...showInputsOnly.optionalProfiles ?? [] ];
-            combinedProfiles.forEach((profile) => {
-                const name = ConfigUtils.getActiveProfileName(profile, commandParameters.arguments); // get profile name
-                const props = this.mConfig.api.secure.securePropsForProfile(name); // get secure props
-                configSecureProps.push(...props); // add to list
-            });
-        }
 
         /**
-         * Determine if Zowe Team Config is in effect.  If it is, then we will construct
-         * a Set of secure fields from its API.  If it is not, then we will construct
-         * a Set of secure fields from the `ConnectionPropsForSessCfg` defaults.
+         * Construct a Set of secure fields from Zowe Team Config API.
          */
-        const secureInputs: Set<string> =
-        useConfig ?
-            new Set([...configSecureProps]) :
-            new Set([...LoggerUtils.CENSORED_OPTIONS, ...LoggerUtils.SECURE_PROMPT_OPTIONS]);
```

Comment:
Related to my previous comment - is this change deliberate? I think we'd want to continue censoring options defined in `LoggerUtils.CENSORED_OPTIONS` and `LoggerUtils.SECURE_PROMPT_OPTIONS`. If someone defines user/password in plain text in their config and omits it from the secure array, we may still want to hide it.

Reply:
deliberate. from my understanding this is determining how to omit from the secure array, if useConfig that means teamConfig and to use first method, otherwise you're using v1 and need to use the latter method

Reply:
>If someone defines user/password in plain text in their config and omits it from the secure array, we may still want to hide it.

🤔 I would not mind to see user and password in plain text (if I'm running things in plain text), and may even be slightly confused about the user and password not showing up in those scenarios

Reply:
This seems like potentially a breaking change, but I understand the reasoning behind it if we believe that users should have control over which fields are secure. 👍  For v1 profiles there was no equivalent to the secure array so we had to hardcode a list of options, but now we don't anymore 😋